adcm_aio_client/core/objects/cm.py

```diff
@@ -397,17 +393,12 @@ async def add(self: Self, host: Host | Iterable[Host] | None = None, filters: Fi
     async def remove(self: Self, host: Host | Iterable[Host] | None = None, filters: Filter | None = None) -> None:
         hosts = await self._get_hosts_from_arg_or_filter(host=host, filters=filters)
 
-        results = await asyncio.gather(
-            *(self._requester.delete(*self._path, host_.id) for host_ in hosts), return_exceptions=True
+        errors = await safe_gather(
+            coros=(self._requester.delete(*self._path, host_.id) for host_ in hosts), objects=hosts
```

Comment:
I suggest making `safe_gather` universal and just returning bunch of exceptions -- let user work with them

Since host may be iterable => can be generator => can "worn out" during  `for host_ in hosts`

Reply:
reworked